crates/services/txpool_v2/src/tests/pool.rs

```diff
@@ -1468,7 +1468,7 @@ async fn insert_single__blob_tx_fails_if_blob_already_exists_in_database() {
 }
 
 #[tokio::test]
-async fn remove_dependency_chain_on_insertion() {
+async fn tx3_dependent_and_conflicts_with_tx2() {
```

Comment:
I think the SUT here would be `insert`. You okay with naming it something like

```rs
async fn insert__if_tx3_depends_on_input_in_tx2_then_tx3_fails()
```

or something like that? Is that a good description?

Reply:
Mhh I applied your suggestion with some modification to be more explicit on the "dependent and collides" and remove the "ten fails" because has I remember the "__" after the insert should be read as "insert fails..." and so the info is already there.